asllib/doc/AbstractSyntax.tex

```diff
@@ -935,4 +935,4 @@ \section{Abstract Syntax Abbreviations\label{sec:AbstractSyntaxAbreviations}}
 % \begin{itemize}
 % \item ASTRule.StmtFromList (see \secref{ASTRule.StmtFromList})
 % \item ASTRule.SequenceStmts (see \secref{ASTRule.SequenceStmts})
-% \end{itemize}
\ No newline at end of file
+% \end{itemiz}
```

Comment:
This is commented out so it doesn't matter much, but it's better to undo this change, which, if uncommented, won't build.

Reply:
Good spot, I didn't see the missing character, I thought my editor just added a trailing newline there 😅 my bad patch incoming.